src/app/organization/working-days/working-days.component.ts

```diff
@@ -0,0 +1,119 @@
+/** Angular Imports */
+import { Component, OnInit } from '@angular/core';
+import { FormGroup, FormBuilder, FormArray, FormControl } from '@angular/forms';
+import { Router, ActivatedRoute } from '@angular/router';
+
+/** Custom Services */
+import { OrganizationService } from '../organization.service';
+
+/** Recurrence default value. */
+const recurrenceDefaultValue = 'FREQ=WEEKLY;INTERVAL=1;BYDAY=';
+
+/**
+ * Working days component.
+ */
+@Component({
+  selector: 'mifosx-working-days',
+  templateUrl: './working-days.component.html',
+  styleUrls: ['./working-days.component.scss']
+})
+export class WorkingDaysComponent implements OnInit {
+
+  /** Working days form. */
+  workingDaysForm: FormGroup;
+  /** Working days data. */
+  workingDaysData: any;
+  /** Week days */
+  weekDays = [
+    {name: 'Monday', value: 'MO', checked: false},
```

Comment:
```suggestion
    { name: 'Monday', value: 'MO', checked: false },
```
Please add one space between the text and the brackets everywhere.